### PR TITLE
Add endpoint for approve workflow run

### PR DESCRIPTION
**Pull request by Nyholm**
See https://docs.github.com/en/rest/reference/actions#approve-a-workflow-run-for-a-fork-pull-request

### CODE REVIEW

File: test/Github/Tests/Api/Repository/Actions/WorkflowRunsTest.php [hunk 1]
```diff
@@ -198,6 +198,24 @@ public function shouldDeleteWorkflowRunLogs()
...
+
+        /** @var WorkflowRuns|MockObject $api */
+        $api = $this->getApiMock();
+
+        $api->expects($this->once())
+            ->method('post')
+            ->with('/repos/KnpLabs/php-github-api/actions/runs/374473304/approve')
+            ->will($this->returnValue($expectedValue));
+
+        $this->assertEquals($expectedValue, $api->approve('KnpLabs', 'php-github-api', 374473304));
```
Comment (dereuromark) [inline test/Github/Tests/Api/Repository/Actions/WorkflowRunsTest.php:216]: You can use assertSame() here.

Comment (Nyholm) [reply test/Github/Tests/Api/Repository/Actions/WorkflowRunsTest.php:216]: Thank you

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thank you @Nyholm

**Comment by Nyholm** [comment]
Awesome. Thank you. 

May I ask for a release within the next few days?

**Comment by acrobat** [comment]
@Nyholm I've tagged [3.3.0](https://github.com/KnpLabs/php-github-api/releases/tag/v3.3.0)!

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**